core/modules/qdisp/Executive.cc

```diff
@@ -470,7 +463,6 @@ void Executive::_waitUntilEmpty() {
         count = _requesters.size();
         _reapRequesters(lock);
         if(count != lastCount) {
-            lastCount = count;
```

Comment:
Multiple threads are at work here. The message will always show the first time through, as lastCount = -1, but the _requesters.size can be changed by other threads, and that will trigger the log message to be printed.